update launch web modeler page (#4214)
camunda/camunda-docs: docs/components/modeler/web-modeler/launch-web-modeler.md

---
id: launch-web-modeler
title: Launch Web Modeler
description: "Design and implement your first diagram using Web Modeler, a component of Camunda where you can also create BPMN diagrams, DMN diagrams, or forms."
---

<span class="badge badge--cloud">Camunda 8 only</span>

> Design and implement your first diagram using Web Modeler, a component of Camunda where you can also create BPMN diagrams, DMN diagrams, or forms.

Reviewer: ```suggestion
Design and implement your first diagram using Web Modeler, a component of Camunda, where you can also create BPMN diagrams, DMN diagrams, or forms.
```
My brain wants an additional comma here. WDYT? If so, please also add it to the metadata description.

Reviewer 2: I get this sense also - sometimes when I can't decide which version is easier to read, I'll just make two sentences for clarity?
```suggestion
Design and implement your first diagram using Web Modeler, a component of Camunda. Web Modeler also allows you to create BPMN diagrams, DMN diagrams, and forms.
```

Author: Updated -- thank you both!

To launch Web Modeler, follow the steps below:

1. After [creating an account and logging in](/guides/create-account.md) to Camunda, you are automatically taken to Web Modeler. Navigate between the components by clicking the square-shaped **Camunda components** icon in the top left corner of the page.
2. Select **Create new project** to create a new project and store diagrams.
   ![web modeler empty home](img/web-modeler-new-user-home.png)
3. Name your diagram. You can go back and change the name any time by clicking on the project name and **Edit name**.

> 4. Select **Browse blueprints** to view blueprints for various use cases as a starting point for your first diagram. Open these blueprints by selecting **Use Blueprint**. Alternatively, click **Create new > BPMN diagram** to create a blank BPMN diagrams.

Reviewer: Is it "blueprints" or "Blueprints" in the product? The casing should be consistent, but it's if inconsistent across the product let's be consistent **with** the product until that's changed.

Author: This is indeed how it is referenced in the product currently with the alternating casing -- happy to raise it with the Modeler team as far as UI goes, but matching it in the docs for now.

Reviewer: Thank you! Please raise it with them.
